Terraform.gitignore

```diff
@@ -32,3 +32,6 @@ override.tf.json
 # Ignore CLI configuration files
 .terraformrc
 terraform.rc
+
+# Ignore hcl file
+*.hcl
```

Comment:
Could we call out the `.terraform.lock.hcl` file specifically?

If not, could you provide more external references to justify the more broad rule?

Reply:
yes we can call it as `.terraform.lock.hcl` instead of wildcard expresssion `*.hcl`.

shall i raise a new PR? please advice

Reply:
Updating this PR with the change is preferred in this situation. Thank you!